### DISCUSSION TITLE
MySQLConnection.cpp

### DISCUSSION THREAD

**Discussion by windild** [azerothcore/azerothcore-wotlk#20372] [General]
when i  use "git pull" command to update the codes and make them, i found this:
/opt/ac/AzerothCore-wotlk-with-NPCBots/src/server/database/Database/MySQLConnection.cpp:132:9: fatal error: use of undeclared identifier 'mysql_ssl_mode'; did you mean 'mysql_ssl_set'?
        mysql_ssl_mode opt_use_ssl = SSL_MODE_DISABLED;
        ^~~~~~~~~~~~~~
        mysql_ssl_set
/usr/include/mariadb/mysql.h:539:14: note: 'mysql_ssl_set' declared here
int             STDCALL mysql_ssl_set(MYSQL *mysql, const char *key,
                        ^
1 error generated.
who can tell me how to  fix it? thanks a lot.

**Comment by windild** [comment]
+---------------------------+
| VERSION()                 |
+---------------------------+
| 10.11.6-MariaDB-0+deb12u1 |
+---------------------------+

**Comment by windild** [comment]
mysql  Ver 15.1 Distrib 10.11.6-MariaDB, for debian-linux-gnu (x86_64) using  EditLine wrapper

**Comment by windild** [comment]
gcc (Debian 12.2.0-14) 12.2.0

**Comment by spacefox-platforms** [comment]
was running into the same issue and made [this](https://github.com/azerothcore/azerothcore-wotlk/issues/20373)
mariaDB is no longer supported, just mysql 8.0 and 8.4

**Reply by windild** [reply]
OK,i see , thank you!
